.changeset/forty-sheep-refuse.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-react-components-development-kitchen': patch
```

Comment:
As this is a visual change that _could_ affect layout it feels like it should be a minor change at least?

Reply:
its a fix not a change, no?

Reply:
agree with james, you can see the impact in the diff 
<img width="347" alt="image" src="https://github.com/guardian/csnx/assets/867233/8f31cf91-b479-411a-af49-60a82ad8007b">